airbyte-integrations/connectors/source-s3/pyproject.toml

```diff
@@ -37,4 +42,6 @@ moto = "==4.2.14"
 docker = "^7.0.0"
 pytest-mock = "^3.6.1"
 requests-mock = "^1.9.3"
-pandas = "^2.0.3"
```

Comment:
Intentional?

Reply:
Yes. The pandas pin was a hack because --use-local-cdk doesn't properly pin/re-pin dependencies, which forces connector maintainers to "fake it" by forcing their dependencies list to deterministically resolve to the same thing that their local CDK resolves to.

We should consider updating `airbyte-ci` to re-lock dependencies when using `--use-local-cdk`, as this will reduce the need to redundantly pin dependencies in the connector when consuming the local CDK.